autoload/lsp/hover.vim

```diff
@@ -72,12 +72,14 @@ enddef
 
 # process the 'textDocument/hover' reply from the LSP server
 # Result: Hover | null
-export def HoverReply(lspserver: dict<any>, hoverResult: any): void
+export def HoverReply(noEcho: bool, lspserver: dict<any>, hoverResult: any): void
   var [hoverText, hoverKind] = GetHoverText(lspserver, hoverResult)
 
   # Nothing to show
   if hoverText->empty()
-    util.WarnMsg($'No hover messages found for current position')
+    if !noEcho
+      util.InfoMsg($'No hover messages found for current position')
```

Comment:
I think this should stay as a warning, as to be consistent with all the other messages:

    util.WarnMsg('Error: No alternate file found')
    util.WarnMsg('Error: No references found')
    ...

Reply:
i do not think it is a warn, but just a regular message, but ok, it's not a matter. modified.